astropy/io/ascii/misc.py

```diff
@@ -0,0 +1,124 @@
+"""A Collection of useful miscellaneous functions.
+
+misc.py:
+  Collection of useful miscellaneous functions.
+
+:Copyright: None -- Hack away at your leisure!!
```

Comment:
@apodemus - sorry for getting to this very late.  Can you add the line below to the top of `misc.py` and remove the `Copyright: None ...` line.  For the astropy project source code we want to have a uniform license including relevant copyright restrictions.
```
# Licensed under a 3-clause BSD style license - see LICENSE.rst
```